ui/src/main/java/edu/wpi/grip/ui/pipeline/input/TextFieldInputSocketController.java

```diff
@@ -0,0 +1,56 @@
+package edu.wpi.grip.ui.pipeline.input;
+
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.assistedinject.Assisted;
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.events.SocketChangedEvent;
+import edu.wpi.grip.ui.pipeline.SocketHandleView;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.scene.control.TextField;
+
+import javax.inject.Inject;
+
+/**
+ * An {@link InputSocketView} that lets the user set the value of a number with a {@link javafx.scene.control.TextField}
+ */
+public class TextFieldInputSocketController extends InputSocketController<String> {
+
+    @Inject private EventBus eventBus;
```

Comment:
This will screw up PMD.
They need to be on separate lines.
Also, it isn't even used in this class so it doesn't need to be injected.